curvefs/src/tools/copyset/curvefs_copyset_base_tool.h

```diff
@@ -31,13 +31,13 @@
 #include "curvefs/proto/mds.pb.h"
 #include "curvefs/proto/topology.pb.h"
 #include "curvefs/src/mds/topology/deal_peerid.h"
-#include "curvefs/src/tools/check/curvefs_copyset_check.h"
+#include "curvefs/src/tools/copyset/curvefs_copyset_status.h"
 
 namespace curvefs {
 namespace tools {
 namespace copyset {
 
-uint64_t GetCopysetKey(uint64_t copysetId, uint64_t poolId);
+uint64_t GetCopysetKey(const uint64_t& copysetId, const uint64_t& poolId);
```

Comment:
GetCopysetKey(uint64_t copysetId, uint64_t poolId) is enough.

Reply:
> GetCopysetKey(uint64_t copysetId, uint64_t poolId) is enough.

fix